app/Models/Film.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Film extends Model
+{
+    use HasFactory;
+    public function videoname()
+    {
+        return $this->hasMany(FilmVideoName::class);
+    }
+    public function alternativname()
+    {
+        return $this->hasMany(FilmAlternativName::class);
+    }
+    public function gender()
```

Comment:
I think genre is more suitable name for this method.

Reply:
I once corrected him. But looks as it remained just in table names.

Reply:
changed thanks!